src/test/java/reactor/kafka/AbstractKafkaTest.java

```diff
@@ -54,17 +54,22 @@
 import org.junit.rules.TestName;
 
 import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.TestcontainersConfiguration;
 import reactor.core.publisher.Flux;
 import reactor.kafka.receiver.ReceiverOptions;
 import reactor.kafka.sender.SenderOptions;
 import reactor.kafka.sender.SenderRecord;
 import reactor.kafka.util.TestUtils;
 
 public abstract class AbstractKafkaTest {
+    private static final String KAFKA_DOCKER_TAG = "5.5.1";
```

Comment:
could you please revert changes in this file? These are unrelated to the Kafka client update

Reply:
I could revert, but my motivation was to enable testing against the recent version, and `public KafkaContainer()` is deprecated.

Reply:
When updating dependencies, it is better to not change any unrelated code to avoid incompatibilities.
Feel free to submit a separate PR that polishes the `KafkaContainer` usage (and can be discussed separately) 👍

Reply:
extracted into a separate PR https://github.com/reactor/reactor-kafka/pull/182